tripal/includes/TripalEntityCollection.inc

```diff
@@ -528,6 +528,7 @@ class TripalEntityCollection {
     if ($job) {
       $job->setTotalItems($total_entities);
     }
+    $job->setInterval(1);
```

Comment:
shouldnt this also be in the `if($job)` check?

Reply:
Oh good catch!  Thanks @bradfordcondon I fixed it.